tests/main/special-home/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Check that Jenkins user can run snaps
+systems: [ubuntu-16.04-64]
+environment:
+    SPECIAL_USER_NAME/jenkins: jenkins
+prepare: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-sh
+    # Install the corresponding package that brings the special user account.
+    case "$SPECIAL_USER_NAME" in
```

Comment:
Why not just "apt install $SPECIAL_USER_NAME" :) ?

Reply:
haha, it might just work :)